writing_guides.md

```diff
@@ -0,0 +1,11 @@
+### Writing this Guide
+
+Docs are written in Mardkown. See e.g. this nice [guide](https://guides.github.com/features/mastering-markdown/).
```

Comment:
Prefer "Good examples can be found in this [guide](https://guides.github.com/features/mastering-markdown/)."